### ISSUE TITLE
Intent explanation nr. 5 is 'area' based, must be 'size' based 2.5.8: Target Size (Minimum)

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2432]
Intent explanation nr. 5 is 'area' based, must be 'size' based 2.5.8: Target Size (Minimum)
https://w3c.github.io/wcag/understanding/target-size-minimum.html

> In cases where a target does not have an area equivalent to 24 by 24 CSS pixels, but there is another control that can achieve the underlying function that does meet the minimum area, the target can be excepted based on the "Equivalent" exception.

See also: https://github.com/w3c/wcag/issues/2426

**Comment by alastc** [comment]
Added to #3012
